cluster/manifests/fabric-gateway/deployment.yaml

```diff
@@ -8,13 +8,7 @@ metadata:
     application: gateway-operator
     component: fabric-gateway-controller
     version: "{{ $version }}"
-    mode: "{{ .Cluster.ConfigItems.fabric_gateway_controller_mode }}"
 spec:
-{{ if eq .Cluster.ConfigItems.fabric_gateway_controller_mode "disabled" }}
-  replicas: 0
-{{ else }}
-  replicas: 1
```

Comment:
How many replicas there will be if spec.replicas is not specified?

Reply:
0

Reply:
changed to have the field but it actually defaults to 1